<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametryBadaniaCzujkiDlg</class>
 <widget class="QWidget" name="ParametryBadaniaCzujkiDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Identyfikacja</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lrodzajsystemu">
        <property name="text">
         <string>Rodzaj systemu:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Odbiornik + Nadajnik</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nadajnik-obiornik + Reflektor</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lproducent">
        <property name="text">
         <string>Producent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="producent"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ltypTransmitter">
        <property name="text">
         <string>Typ nadajnika</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="typTransmitter"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ltypReceiver">
        <property name="text">
         <string>Typ odbiornika</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="typReceiver"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lrozstawienieminimalne">
        <property name="text">
         <string>Rozstawienie minimalne</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="rozstawienieMinimalne"/>
        </item>
        <item>
         <widget class="QLabel" name="lunitrozmin">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lrozstawieniemaksymalne">
        <property name="text">
         <string>Rozstawienie maksymalne</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="rozstawienieMaksymalne"/>
        </item>
        <item>
         <widget class="QLabel" name="lunitrozmax">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="line_1">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>4</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="4" column="0">
          <widget class="QLabel" name="lospionowa">
           <property name="text">
            <string>Oś pionowa</string>
           </property>
          </widget>
         </item>
         <item row="5" column="6">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&amp;deg;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="6">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&amp;deg;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="7">
          <widget class="QLabel" name="lcaption">
           <property name="text">
            <string>Maksymalna kątowa niewspółosiowość</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="transmitter_ospozioma"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="lunitpionowa">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&amp;deg;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QLineEdit" name="receiver_ospionowa"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lospozioma">
           <property name="text">
            <string>Oś pozioma</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="transmitter_ospionowa"/>
         </item>
         <item row="2" column="0" colspan="7">
          <widget class="Line" name="line_2">
           <property name="minimumSize">
            <size>
             <width>5</width>
             <height>5</height>
            </size>
           </property>
           <property name="lineWidth">
            <number>4</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="4" rowspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="5" colspan="2">
          <widget class="QLabel" name="lReceiver">
           <property name="text">
            <string>Odbiornik</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QLineEdit" name="receiver_ospozioma"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="lunitospozioma">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&amp;deg;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLabel" name="lTransmitter">
           <property name="text">
            <string>Nadajnik</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Numery fabryczne</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frameCzujki">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayoutNumerCzujek">
         <item row="0" column="3" colspan="2">
          <widget class="QLabel" name="headSortowanie">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Pocz. kol. czujek
Zmierz. czułość</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="hnumerReceiver">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Numer nadajnika</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="hlp">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Nr.
Czujki</string>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="hnumerTransmitter">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Numer odbiornika</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetSortowanie" native="true">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="3" column="0">
          <widget class="QLabel" name="luwaga">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(26, 95, 180)</string>
           </property>
           <property name="text">
            <string>UWAGA</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lcontext">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(26, 95, 180)</string>
           </property>
           <property name="text">
            <string>Czujki zostaną posortowane zgodnie z normą</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lError">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:red</string>
           </property>
           <property name="text">
            <string>UWAGA</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="error">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:red</string>
           </property>
           <property name="text">
            <string>Wartości numerów dla czujki są puste </string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ilość czujek</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="iloscczujek">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lUwaga2">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:magenta</string>
           </property>
           <property name="text">
            <string>UWAGA</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lContext2">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:magenta</string>
           </property>
           <property name="text">
            <string>Zgodnie z normą ilość czujek
powinna być równa 7</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>producent</tabstop>
  <tabstop>typTransmitter</tabstop>
  <tabstop>typReceiver</tabstop>
  <tabstop>rozstawienieMinimalne</tabstop>
  <tabstop>rozstawienieMaksymalne</tabstop>
  <tabstop>transmitter_ospionowa</tabstop>
  <tabstop>transmitter_ospozioma</tabstop>
  <tabstop>receiver_ospionowa</tabstop>
  <tabstop>receiver_ospozioma</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
